go.mod

```diff
@@ -61,6 +64,7 @@ require (
 replace (
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20191219222812-2987a591a72c
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20191125132246-f6563a70e19a
+	github.com/openshift/library-go => github.com/mhenriks/library-go v0.0.0-20200116194830-9fcc1a687a9d
```

Comment:
using fork until this is merged:  https://github.com/openshift/library-go/pull/540

Reply:
If we merge this could you create a gh issue so we don't forget to back to the trunk?